tests/integration-async/records/conftest.py

```diff
@@ -127,7 +127,10 @@ def _check(steps={}, timeout=15):
                 _expected_result = step.get("expected_result")
                 _fun = step.get("step")
                 _expected_key = None
-                result = _fun(*_args, **_kwargs)
+                try:
+                    result = _fun(*_args, **_kwargs)
+                except:
+                    pass
```

Comment:
what's going on here?

Reply:
This ```retry_until_matched``` wasn't actually retrying when we were checking if a record was in ES because we were getting an error and it was exiting. This is also why we had all those time.sleep in the tests that were actually defeating the purpose of the ```retry_until_matched```